solvers/DeepCORAL.py

```diff
@@ -49,7 +50,6 @@ def get_estimator(self, n_classes, device, dataset_name, **kwargs):
         net = DeepCoral(
             model,
             optimizer=AdamW,
-            reg=1,
```

Comment:
`reg` is the DA parameter (the docs is not correct in skada). In this commit, you just removed it, but you didn't add it in the grid search, right?

Reply:
Oh my bad, it thought it was an old optimiser reg term